tests/stress_tests/scripts/compare_memcheck_2_runs.py

```diff
@@ -168,7 +168,12 @@ def cli_parser():
     parser.add_argument('--db_url',
                         help='MongoDB URL in a for "mongodb://server:port".')
     parser.add_argument('--db_collection',
-                        help=f'Collection name in {DATABASE} database to query.',
+                        help=f'Collection name in "{DATABASE}" database to query'
+                             f' data using current source.',
+                        choices=["commit", "nightly", "weekly"])
```

Comment:
Allowed collections list is used twice. We should define a constant for it:

https://github.com/openvinotoolkit/openvino/blob/master/tests/stress_tests/scripts/run_memcheck.py#L24

Reply:
I will do it in scope of https://github.com/openvinotoolkit/openvino/pull/1036